web/e2e/pages/landing/LandingPage.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import type { WebDriver, WebElement } from 'selenium-webdriver'
+import { By } from 'selenium-webdriver'
+
+export default class LandingPage {
+  driver: WebDriver
+  language: string
+
+  constructor (driver: WebDriver, langugage: 'de' | 'en' = 'de') {
```

Comment:
Why should this just be `de` and `en`

Reply:
I think this is fine for now as this task is only about the setup.